src/animation/shine.js

```diff
@@ -11,14 +11,13 @@ const styles = StyleSheet.create({
     width: 30,
     position: 'absolute',
     height: '100%',
-    backgroundColor: '#ffffff',
     opacity: 0.4,
   },
 });
 /**
  * Create a repetitive Shine animation
  */
-const Shine = ({ children }) => {
+const Shine = ({ backgroundColor, children }) => {
```

Comment:
Can you rename it `animationColor` ? It makes more sense to be there. WYT?

Reply:
Yeah, it's make more sense. I agree 100%